packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -161,7 +161,9 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
       showUnknownPersonaCoin
     });
 
-    return (
+    // return Image if imageUrl is present
+    // this check will stop from rendering an empty img tag
```

Comment:
Some typos. What about:
"Render the Image component only if an image URL is provided"?